distributed/tests/test_profile.py

```diff
@@ -46,6 +47,43 @@ def test_f():
     assert 95 < g['count'] + h['count'] <= 100
 
 
+@pytest.mark.skipif(
+        sys.version_info < (3, 5),
+        reason='stacktrace not supported on python < 3.5')
+@pytest.mark.skipif(
+        WINDOWS,
+        reason='no low-level profiler support for Windows available')
+def test_basic_low_level():
+    def test_g():
+        time.sleep(0.05)
+
+    def test_f():
+        for i in range(100):
+            test_g()
+
+    thread = threading.Thread(target=test_f)
+    thread.daemon = True
+    thread.start()
+
+    state = create()
+
+    for i in range(100):
+        time.sleep(0.02)
+        frame = sys._current_frames()[thread.ident]
+        llframes = {thread.ident: ll_get_stack(thread.ident)}
+        for f in llframes.values():
+            if f is not None:
+                llprocess(f, None, state)
+
+    assert state['count'] == 100
+    d = state
+    while len(d['children']) == 1:
+        d = first(d['children'].values())
+
+    assert d['count'] == 100
+    assert '__restore_rt' or '_L_unlock' in str(d['description'])
```

Comment:
This assert is always True.  

```
In [5]: 'a' or 0 in []
Out[5]: 'a'
```

it should be `assert '__restore_rt' in str(d['description']) or '_L_unlock' in str(d['description'])`

Reply:
Ah, right, thanks for spotting it!